src/code/z_fbdemo.c

```diff
@@ -1,20 +1,118 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/func_800B18B0.s")
+extern u16 COLOR_BUFFER_SEG[];
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/func_800B1CFC.s")
+Gfx D_8012AFB0[] = {
+    gsDPPipeSync(),
+    gsDPSetCycleType(G_CYC_FILL),
+    gsDPSetColorImage(G_IM_FMT_RGBA, G_IM_SIZ_16b, 320, COLOR_BUFFER_SEG),
+    gsDPSetFillColor(RGBA8(0x42, 0x11, 0x42, 0x11)),
+    gsDPFillRectangle(0, 0, 319, 239),
+    gsDPPipeSync(),
+    gsDPSetFillColor(RGBA8(0x42, 0x3F, 0x42, 0x3F)),
+    gsDPFillRectangle(20, 20, 300, 220),
+    gsDPPipeSync(),
+    gsSPEndDisplayList(),
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/func_800B1DBC.s")
+Gfx D_8012B000[] = {
+    gsDPPipeSync(),
+    gsSPTexture(0x8000, 0x8000, 0, G_TX_RENDERTILE, G_ON),
+    gsSPClearGeometryMode(G_ZBUFFER | G_SHADE | G_CULL_BOTH | G_FOG | G_LIGHTING | G_TEXTURE_GEN |
+                          G_TEXTURE_GEN_LINEAR | G_LOD | G_SHADING_SMOOTH),
+    gsDPSetCombineMode(G_CC_DECALRGB, G_CC_DECALRGB),
+    gsDPSetOtherMode(G_AD_DISABLE | G_CD_MAGICSQ | G_CK_NONE | G_TC_FILT | G_TF_BILERP | G_TT_NONE | G_TL_TILE |
+                         G_TD_CLAMP | G_TP_PERSP | G_CYC_1CYCLE | G_PM_1PRIMITIVE,
+                     G_AC_NONE | G_ZS_PIXEL | G_RM_AA_OPA_SURF | G_RM_AA_OPA_SURF2),
+    gsSPEndDisplayList(),
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/func_800B1E84.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/FBFilter_InitGraphics.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/func_800B2074.s")
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/FBFilter_InitData.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo/func_800B2188.s")
+void FBFilter_Destroy(FBFilter* this) {
```

Comment:
Based on the discussions we had about it, this system and struct should use the name `TransitionUnk` or whatever.
```suggestion
void TransitionUnk_Destroy(TransitionUnk* this) {
```